src/Molecules/RangeSlider/RangeSlider.tsx

```diff
@@ -17,6 +17,23 @@ const Container = styled.div<InternalProps>`
   margin: 0 ${(p) => `${getKnobSize(p.$variant) / 2}px`};
 `;
 
+const getDefaultKnob = (
+  low: number | undefined,
+  high: number | undefined,
+  min: number,
+  max: number,
+) => {
+  if (low && high && low === high) {
+    const distanceToMin = low - min;
+    const distanceToMax = max - high;
```

Comment:
you might wanna use the absolute value of the diff instead to avoid weird negative values
```suggestion
    const distanceToMin = Math.abs(low - min);
    const distanceToMax = Math.abs(max - high);
```